lib/ansible/modules/network/nxos/nxos_vrf_af.py

```diff
@@ -61,6 +83,46 @@
     afi: ipv4
     route_target_both_auto_evpn: True
     state: present
+
+- nxos_vrf_af:
+    vrf: ntc
+    afi: ipv4
+    route_target_import:
+      - rt: '65000:1000'
+      - rt: '65001:1000'
+
+- nxos_vrf_af:
+    vrf: ntc
+    afi: ipv4
+    route_target_import:
+      - rt: '65000:1000'
+      - rt: '65001:1000'
+        state: absent
+
+- nxos_vrf_af:
+    vrf: ntc
+    afi: ipv4
+    route_target_export:
+      - rt: '65000:1000'
+      - rt: '65001:1000'
+
+- nxos_vrf_af:
+    vrf: ntc
+    afi: ipv4
+    route_target_export:
+      - rt: '65000:1000'
+        state: absent
+
+- nxos_vrf_af:
+    vrf: ntc
+    afi: ipv4
+    route_target_both:
+      - rt: '65000:1000'
+        state: present
```

Comment:
How does the per route_target `state` work with the overall `state` property?  For example what should I expect in the following case:

```yaml
- nxos_vrf_af:
    state: absent
    vrf: ntc
    afi: ipv4
    route_target_export:
      - rt: '65000:1000'
        state: present
```

Reply:
Good point. 

Route-targets exist under the address-family, therefore they can't exists without the address-family. In case of an `absent` address-family, the route-target part (same with `route_target_both_auto_evpn`) is ignored.

Reply:
@richardstrnad I suspected that was the desired workflow so thanks for confirming.  Could you add a note in the doc header to clarify?